manifests/database.pp

```diff
@@ -14,6 +15,12 @@
       before   => Pulpcore::Admin['migrate --noinput'],
     }
 
+    postgresql::server::extension { "hstore for ${pulpcore::postgresql_db_name}":
```

Comment:
It looks like Pulp creates the extension here in their migration: https://github.com/pulp/pulpcore/pull/3427/files#diff-6bc6ee115c471f5d7f62c7cabd997b6197ae986c46cff92096f43b8264507a2dR40

With `HStoreExtension()`

But I'm guessing it doesn't hurt to have it in two places!

Reply:
I went with this since the issue mentioned that it needs super user privileges on PG < 13. We're on PG 12 now.

Having it in the Django migrations will also make sure it fails if it isn't present, and do the right thing on newer PG.